koji_wrapper/factory.py

```diff
@@ -21,7 +35,18 @@ def load_profile(cls, profile='koji', user_config=None):
     @classmethod
     def open_session(cls, profile=None, user_options=None,
                      authenticate=False, debug=False):
+        """open a koji ClientSession using profile or user_options provided
+
+        :param profile koji profile to load (optional)
```

Comment:
I think all the params need a colon after the param name to format correctly when docs are generated, at least that is what I have observed from my own additions.